ui-library-filesharing-chat-composite/api/functions/uploadFileToAzureBlobStore.ts

```diff
@@ -59,7 +59,7 @@ const httpTrigger = async function (req: HttpRequest, context: InvocationContext
     const fileContentBuffer = Buffer.from(fileContent);
     const size = fileContentBuffer.byteLength;
 
-    const uniqueFileName = `${v4()}-${file.name}`;
+    const uniqueFileName = `${v4()}-${CSS.escape(file.name)}`;
```

Comment:
Can JS escape function be used here instead?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape